CHANGELOG.md

```diff
@@ -76,6 +76,11 @@
   See `--sdg-scale-factor` for an updated option providing similar
   functionality.
 
+### Fixes
+
+* `iab config`: Fixed a bug where `ilab` didn't recognize `train.lora_quantize_dtype: null` as a valid
```

Comment:
```suggestion
* `ilab config`: Fixed a bug where `ilab` didn't recognize `train.lora_quantize_dtype: null` as a valid
```

Reply:
As a valid what?

Reply:
probably meant `as valid`

Reply:
`as a valid value`, it's on the next line.

Reply:
https://github.com/RobotSail/instructlab/blob/ef31526b308bb353689a08c03a5b8098a4a9b449/CHANGELOG.md#L82